tests/adjoint_born_test.py

```diff
@@ -14,13 +14,17 @@
 
 
 class AdjointBornLayer(torch.nn.Module):
+    """
+    Creates a wrapped forward-born operator
+    """
     def __init__(self, model, geometry, device):
         super(AdjointBornLayer, self).__init__()
         self.forward_born = devito_wrapper.AdjointBorn()
         self.model = model
         self.geometry = geometry
         self.device = device
-        self.solver = AcousticWaveSolver(self.model, self.geometry, space_order=8)
+        self.solver = AcousticWaveSolver(self.model, self.geometry,
+                                         space_order=4)
```

Comment:
bit low, would keep 8 to keep dispersion fairly low

Reply:
Okay.